.gitpod.yml

```diff
@@ -5,8 +5,11 @@ ports:
       onOpen: open-preview
 tasks:
     - init: >
+        export CHROME_BIN=/usr/bin/chromium-browser &&
+        gem install bundler &&
         bundle install --without production &&
         cp config/database.yml.gitpod config/database.yml &&
+        mysql -uroot -proot -e "SET @@global.sql_mode=(SELECT REPLACE(@@global.sql_mode, 'ONLY_FULL_GROUP_BY', ''));" &&
         mysql -e "CREATE DATABASE plots;" &&
```

Comment:
seeing `App 1912 output: Error: The application encountered the following error: Unknown database 'plots' (ActiveRecord::NoDatabaseError)` here... :-(

Reply:
aha - we got 

`mysql: [Warning] Using a password on the command line interface can be insecure.
ERROR 1045 (28000): Access denied for user 'root'@'localhost' (using password: YES)`

Reply:
I'll try removing the root lines... dunno...